ql/src/test/org/apache/hadoop/hive/ql/metadata/TestSessionHiveMetastoreClientGetPartitionsTempTable.java

```diff
@@ -123,13 +123,13 @@ public void testGetPartitionsByNamesEmptyParts() throws Exception {
     getClient().getPartitionsByNames(DB_NAME, TABLE_NAME, Lists.newArrayList("", ""));
   }
 
-  @Test(expected = MetaException.class)
+  @Test
```

Comment:
This is test was introduced to check backward compatibility of the HMS API.
Changing this behavior indicates change in the API. I think we should keep the original Exception.

Reply:
I don't recall why we (Ramesh and I) made the change in downstream here, but I will see if it can be reverted. I didn't change HMS API per se. The MetaException could be thrown from getPartitionsByNames before my changes. Those are tests only. 
Any way, let me see if I can change to the way the tests were ,and if not, will give justification.

Reply:
It seemed ok to remove and unit test still ok.